app/models/asset_field_type.rb

```diff
@@ -74,6 +75,16 @@ def allowed_content_types
     end
   end
 
+  def asset_extension
+    unless existing_data['asset']['content_type'] == asset_content_type
+     errors.add(:asset, "Asset must be the same type of: #{existing_data['asset']['content_type']}")
+    end
+  end
+
+  def asset_field_type_id
```

Comment:
Originally before we replaced id's with parameterized Titles updated assets were storing new images on updates since the `AssetFieldType` id would be different each time. This caused a few issues beyond just storing a new set of images on each update. 

But it's purpose is less important now that we use Asset Title instead of ID.